src/main/java/uk/gov/hmcts/reform/coh/controller/onlinehearing/CreateOnlineHearingResponse.java

```diff
@@ -2,16 +2,62 @@
 
```

Comment:
Might be worth checking with John if we need to return the other values (except online hearing ID)